database/education.py

```diff
@@ -1,72 +1,87 @@
-salary = {
-    "classification": "Annual Salary",
-    "question": "What is your net monthly salary?",
-    "description": "Gross income is the total amount you earn (typically over the course of a year) before expenses. Net income is the profit your business earns after expenses ",
-    "information": "Depending on the home price you’re aiming for, you may want to wait a year or two before you apply for a mortgage if you’ve just moved into a higher-paying role. The longer you stay in your higher-paying position, the more your lender may be willing to loan you.",
-    "note":"The amount of money you earn plays a smaller role in getting a mortgage than you might think. ",
-    "source": "https://www.rocketmortgage.com"
-}
 zip = {
     "classification": "ZIP Code",
     "question": "What is the zipcode you'd like to live in?",
     "description":"Property taxes and interest rates can vary by location. Enter the ZIP code where you're looking for homes and we'll automatically add property taxes and interest rates for your area.",
     "information": "A “cost of living analysis” by two large realty companies discovered that living in the suburbs versus living in the city does impact your cost of living – by thousands of dollars. ",
     "note": "The term ZIP is an acronym for Zone Improvement Plan",
-    "source": "http://www.trulia.com/"
+    "source": "http://www.trulia.com/",
+    "order": 1
+}
+credit = {
+    "classification": "Credit Score",
+    "question": "What is your current credit score?",
+    "description": " A high score will give you access to lower interest rates and more lender choices. If you have a low score, you may have trouble getting a loan.",
+    "information": "Your credit score plays a big role in the interest rate you’ll get for your loan.",
+    "note": "Your credit score is a numerical rating that ranges from 300 – 850 and tells lenders how responsible you are when you borrow money. ",
+    "source": "https://www.rocketmortgage.com",
+    "order": 2
+}
+salary = {
```

Comment:
Thank the globs that we finally got this content updated!!!  And thank you @priyapower for securing it!

Reply:
I still believe we can flush these out more. I would like to ask the team to give their input and see if any phrases can be updated